### DISCUSSION TITLE
Shovel God Feature

### DISCUSSION THREAD

**Discussion by Integer404** [winstxnhdw/lc-hax#505] [Ideas]
Make a feature which requires 7 shovels, basically it equips and rapidly drops shovels in a specific position that kinda makes it seem like theres shovel floating around you. Not practical at all but would look fucking cool.

**Comment by winstxnhdw** [comment]
I don't want to add new features that don't really serve more than an aesthetic purpose because it's just going to make this codebase harder to maintain. Feel free to add it yourself though. If you know how to code, you can probably take inspiration from the `bomb` command and implement this yourself.

**Reply by Integer404** [reply]
Well said kind sir 💯
